### PR TITLE
SQL to hide unwanted NE borders

### PR DESCRIPTION
**Pull request by jeffdefacto**
This deletes any NE border feature that matches with the list of ne_ids in the `remove_lines` table.

### CODE REVIEW

**Review by nvkelso — CHANGES_REQUESTED** [review]
These all need to be visible for different audiences (mostly Russia, Turkey, Taiwan), but should be `unrecognized` for the others. 

Can you switch this around so it's modifying the `fclass_*` attributes instead, please?

File: data/apply-ne_disputed_border_suppression.sql [hunk 1]
```diff
@@ -0,0 +1,512 @@
...
+    union
+    select ne_id from south_ossetia
+    union
+    select ne_id from transnistria
+    )),
+
+-- Add Russian disputed view to Nagorno-Karabakh
+ab_do_so_tr_ne_10m_admin_0_boundary_lines_land as (
+update ne_10m_admin_0_boundary_lines_land l set
+            fclass_ru = ''
```
Comment (nvkelso) [inline data/apply-ne_disputed_border_suppression.sql:396]: Is the intent this is "default" which happens to be `unrecognized`?

File: data/apply-ne_disputed_border_suppression.sql [hunk 2]
```diff
@@ -0,0 +1,512 @@
...
+
+    )),
+
+
+-- Add Turkish view to Northern Cyprus
+tr_ne_10m_admin_0_boundary_lines_land as (
+update ne_10m_admin_0_boundary_lines_land l set
+            fclass_tr = 'country'
+where l.ne_id in (
+    select ne_id from nagorno_karabakh
```
Comment (nvkelso) [inline data/apply-ne_disputed_border_suppression.sql:439]: Is this for Northern Cyprus or nagorno_karabakh?

File: data/perform-sql-updates.sh [hunk 3]
```diff
@@ -43,6 +43,10 @@ echo "done."
 echo -e "\nDeleting disputed names"
 psql $PSQLOPTS  $@ -f apply-planet_disputed_area_features_name_suppression.sql
 
+# Deletes some of the Natural Earth dispute borders that we do not want to render
```
Comment (nvkelso) [inline data/perform-sql-updates.sh:46]: This is no longer deleting, please update comment and echo

File: data/apply-ne_disputed_border_suppression.sql (hunk 1 above)
Comment (jeffdefacto) [reply data/apply-ne_disputed_border_suppression.sql:396]: Do we want this to be unrecognized? I thought we wanted to show a disputed border there for russia to show that they acknowledge something existing, but not quite a country.

File: data/apply-ne_disputed_border_suppression.sql (hunk 2 above)
Comment (jeffdefacto) [reply data/apply-ne_disputed_border_suppression.sql:439]: Yeah good call. This isn't really ready for review yet. I'm just testing out the syntax

**Review by travisgrigsby — APPROVED** [review]

**Review by nvkelso — COMMENTED** [review]
I really like the direction of this updated PR.

This does point that we need to do a similar `place=unrecognized`  for OSM boundaries so we have consistency on zoom in. Can be followup PR, just link them together please.

**Review by nvkelso — APPROVED** [review]

**Pull request merged**